src/app/test-record/defects/defects.component.scss

```diff
@@ -0,0 +1,25 @@
+
+.govuk-summary-list {
+
+  .tag-custom {
+    padding: 1px 10px 1px 10px;
```

Comment:
could be short handed with 1px 10px also if this a custom component this will need to ported into the global scss file at the bottom.
We could follow the convention govuk lib uses
// colours

// typography

...

/**
** Components
**/

// tags
your scss tag

Also let's avoid using @extend, it is bad practice in general,
this scopes to the parent component and increase specificity making things very complicated if we want to override styles later or use composition (with scss conventions like BEM for example).
It is a great tool when architecting the scss to make it dry but will require some thorough thinking beforehand.